Makefile

```diff
@@ -9,12 +9,20 @@ CGO_ENABLED ?= 0
 TEMPORAL_ROOT := temporal
 TCTL_ROOT := tctl
 CLI_ROOT := cli
-IMAGE_TAG ?= sha-$(shell git rev-parse --short HEAD)
+
 TEMPORAL_SHA := $(shell sh -c 'git submodule status -- temporal | cut -c2-40')
 TCTL_SHA := $(shell sh -c "git submodule status -- tctl | cut -c2-40")
 
+IMAGE_SHA_TAG ?= sha-$(shell git rev-parse --short HEAD)
+IMAGE_BRANCH_TAG ?= branch-$(shell git rev-parse --abbrev-ref HEAD)
+SAFE_IMAGE_BRANCH_TAG = $(shell echo $(IMAGE_BRANCH_TAG)  | tr 'A-Z' 'a-z' | sed 's/[^a-z0-9._-]/-/g')
```

Comment:
Doing this here to make sure all make invocations do this, regardless where from.

Reply:
oh I see. I wonder if we should even do it in hcl instead? maybe that's too confusing